modules/order/src/Plugin/Field/FieldFormatter/LineItemTable.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\commerce_order\Plugin\Field\FieldFormatter.
+ */
+
+namespace Drupal\commerce_order\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\FormatterBase;
+
+/**
+ * Plugin implementation of the 'line_item_table' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "line_item_table",
+ *   label = @Translation("Line item table"),
+ *   field_types = {
+ *     "entity_reference",
+ *   },
+ * )
+ */
+class LineItemTable extends FormatterBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(FieldItemListInterface $items, $langcode) {
+    $order = $items->getEntity();
+    return [
+      '#type' => 'view',
+      '#name' => 'line_item_summary',
+      '#arguments' => [$order->id()],
+      '#embed' => TRUE,
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function isApplicable(FieldDefinitionInterface $field_definition) {
+    return $field_definition->getTargetEntityTypeId() == 'commerce_order';
```

Comment:
We should restrict further by field name, otherwise it will show up as an option on other entity references, like billing profile.